### PR TITLE
Add cosmos to kube

### PR DESCRIPTION
**Pull request by bigglesandginger**
# Description

Please include a summary of the changes and the related issue.

Resolves #(issue)

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]

File: kubernetes/overlays/prod/overlays/askem-staging/kustomization.yaml [hunk 1]
```diff
@@ -133,3 +134,8 @@ images:
   - name: mira-taskrunner-image
     newName: ghcr.io/darpa-askem/mira-taskrunner
     newTag: 'latest'
+
+  # mira taskrunner
```
Comment (dgauldie) [inline kubernetes/overlays/prod/overlays/askem-staging/kustomization.yaml:138]: copy-paste error

**Review by dgauldie — APPROVED** [review]
other than that little copy/paste error, this looks good.

**Comment by dgauldie** [comment]
you will need to fix some conflicts

**Review by dgauldie — CHANGES_REQUESTED** [review]
Can you add cosmos to the dev deployment now?

**Review by dgauldie — APPROVED** [review]

**Pull request merged**